File: python/student_registration.py
def student_registration_status_checker():
    # Prompt for student name
    name = input('Enter the student name: ')

    # Prompt for number of registered units
    units = int(input('Enter the number of registered units: '))

    # Check registration status
    if units > 7:
        status = 'Overload - Approval Required'
    else:
        status = 'Registration Accepted'

    # Print final summary
    print(f'\nSummary:')
    print(f'Student Name: {name}')
    print(f'Registered Units: {units}')
    print(f'Status: {status}')

# Run the function
student_registration_status_checker()

File: javascript/student_registration.js
// Student Registration Status Checker

function checkStudentRegistration() {
    const studentName = prompt('Enter student name:');
    const registeredUnits = parseInt(prompt('Enter number of registered units:'), 10);
    let status;

    // Check registration status
    if (registeredUnits > 7) {
        status = 'Overload - Approval Required';
    } else {
        status = 'Registration Accepted';
    }

    // Display final summary
    console.log(`\nSummary:\nStudent Name: ${studentName}\nRegistered Units: ${registeredUnits}\nStatus: ${status}`);
}

// Call the function to execute the program
checkStudentRegistration();